src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -76,11 +76,41 @@ def create_webapp(resource_group_name, name,
 
     webapp_def = Site(server_farm_id=plan_id, location=location)
     poller = client.web_apps.create_or_update(resource_group_name, name, webapp_def)
-    return AppServiceLongRunningOperation()(poller)
+    site = AppServiceLongRunningOperation()(poller)
+
+    profiles = list_publish_profiles(resource_group_name, name, None)
+    publishURL="";
+    for profile in profiles:
```

Comment:
Please share the implementations with `show`. Also simplify the code, as I don't think we need 3 nested loop here.
Please re-record the impacted tests